platform/device-manager.ts

```diff
@@ -3,24 +3,18 @@
  * TODO(cjdaly) integrate with Mock Local Home Platform and Mock Network
  **/
 /// <reference types="@google/local-home-sdk" />
+import { MockNetwork } from './mock-radio';
 
-import { DeviceManager } from "@google/local-home-sdk";
-
-export class DeviceManagerStub implements DeviceManager{
-
-  markPending(request: smarthome.IntentRequest): Promise<void> {
-    //TODO(cjdaly) implementation
-    throw new Error("Method not implemented.");
-  }
-
-  getProxyInfo(id: string): smarthome.ProxyInfo {
-    //TODO(cjdaly) implementation
-    throw new Error("Method not implemented.");
-  }
-
+export class DeviceManagerStub implements smarthome.DeviceManager {
   deviceId: string;
   error: boolean;
   commands: smarthome.DataFlow.TcpRequestData[] = [];
+  private mockNetwork: MockNetwork;
+
+  constructor(mockNetwork: MockNetwork) {
+    this.mockNetwork = mockNetwork;
+  }
+
   public send(
```

Comment:
In Typescript, do methods need to be marked explicitly as public?

Reply:
The default modifier is `public`, so I don't really need to, no.  I've been adding it out of habit since I'm not used to TypeScript, but I could remove them (I think there's an eslint rule I can add).  Might slightly help others not used to TypeScript to leave it in.